style.css

```diff
@@ -0,0 +1,143 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  min-height: 100vh;
+  background: #c9c9c9;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
+  font-weight: 400;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.add-task-wrapper {
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.add-input-error-label {
+  color: #fff;
+  position: absolute;
+  top: 0;
+  left: 0;
+  padding: 5px 10px;
+  width: 100%;
+  z-index: -1;
+  transition: 200ms all ease-in-out;
+}
+
+.add-input-error-label.active {
+  top: 38px;
+  color: red;
+}
+
+.add-task-input {
+  outline: none;
+  padding: 10px 15px;
+  border-radius: 3px;
+  border: 1px solid #fff;
+  margin-right: 10px;
+  min-width: 200px;
+}
+
+.add-task-button {
+  padding: 5px 15px;
+  border-radius: 3px;
+  border: none;
+  background-color: green;
+  color: #fff;
+  font-size: 24px;
+}
+
+.add-task-button:hover {
+  background-color: darkgreen;
+  cursor: pointer;
+}
+
+.columns-wrapper {
+  width: 100%;
+  height: 75vh;
+  max-width: 1100px;
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-around;
+  margin-top: 50px;
+}
+
+.tasks-column {
+  padding: 10px;
+  background-color: #fff;
+  width: 30%;
+  border-radius: 5px;
+  overflow: hidden;
+  box-shadow: rgba(0, 0, 0, 0.1) 4px 4px 12px;
+}
+
+.tasks-column-header {
+  padding: 10px;
+  text-align: center;
+  background-color: #ddd;
+}
+
+.single-task {
+  text-align: center;
+  font-size: 20px;
+  margin-top: 15px;
+  padding: 15px 15px 10px;
+  border-radius: 5px;
+  background-color: #f0f0f0;
+}
+
+.task-button-wrapper {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-top: 15px
+}
+
+.button-bin {
+  background-image: url("img/bin.svg");
+  width: 15px;
+  height: 15px;
+  background-size: contain;
+  border: none;
+  background-color: transparent;
+  opacity: 0.4;
+}
+
+.button-right-arrow, .button-left-arrow {
+  background-image: url("img/arrow-right.svg");
+  width: 25px;
+  height: 25px;
+  background-size: contain;
+  border: none;
+  background-color: transparent;
+  opacity: 0.4;
+}
+
+.button-left-arrow {
+  transform: rotate(180deg);
+}
+
+.button-bin:hover,
+.button-right-arrow:hover,
+.button-left-arrow:hover {
+  opacity: 1;
+  cursor: pointer;
+}
+
+.tasks-column:first-of-type .button-left-arrow {
+  opacity: 0;
+  pointer-events: none;
+}
+.tasks-column:last-of-type .button-right-arrow {
+  opacity: 0;
+  pointer-events: none;
+}
```

Comment:
This is a little hacky. Personally, I'd do that through JavaScript to be more explicit.